metagraph/plugins/networkx/algorithms.py

```diff
@@ -132,10 +152,7 @@ def nx_minimum_spanning_tree(graph: NetworkXGraph) -> NetworkXGraph:
 
     @concrete_algorithm("centrality.betweenness")
     def nx_betweenness_centrality(
-        graph: NetworkXGraph,
-        nodes: mg.Optional[PythonNodeSet],
-        normalize: bool,
-        # include_endpoints: bool,
```

Comment:
I think it would be useful to put into the documentation that this algorithm does not include endpoints as part of the counts for a node.

Reply:
I didn't see this in other implementations, so I'm leaving out for now. If it causes confusion for users, we can make a note in future docs.